gatsby-browser.js

```diff
@@ -5,3 +5,4 @@
  */
 
 // You can delete this file if you're not using it
+import "./node_modules/bootstrap/dist/css/bootstrap.css"
```

Comment:
I just realized that because this PR has code from #8 in it, there will be a merge conflict here.